src/views/visualize.hbs

```diff
@@ -1,7 +1,25 @@
-{{>header}}
-
-<h1>THIS IS THE VISUALIZE PAGE</h1>
-<svg width="100%" height="100%"></svg>
+<head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width,
+      initial-scale=1.0">
+      <meta http-equiv="X-UA-Compatible" content="ie=edge">
+      <link rel="stylesheet" href="https://unpkg.com/tachyons@4.9.0/css/tachyons.min.css"/>
+      <link
+      href="https://fonts.googleapis.com/css?family=Roboto"
+      rel="stylesheet">
+      <style type="text/css" media="screen">
+        svg{
+          display: block;
```

Comment:
Is this the right place to put this?

Reply:
this block was moved to CSS file in next PR, so for now I just deleted it.